CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog for AWS IoT Jobs Library
 
+## v1.4.0 (November 2023)
+- [#90](https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/pull/90) Update doxygen to reflect addition of OTA Job Parser
+- [#88](https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/pull/88) ADD OTA Job Parser from [Labs-Project-example-for-aws-iot-core](https://github.com/FreeRTOS/Labs-Project-ota-example-for-aws-iot-core)
```

Comment:
I guess we should not mention Labs-Project-ota-example-for-aws-iot-core in the changelog. Just give a short description of what we are adding in the PR.